payment-instruments/mobile-pay/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: MobilePay Online Payments
+redirect_from: /payments/mobile-pay/
 estimated_read: 30
 description: |
   MobilePay is the fast and simple way of
```

Comment:
```suggestion
  MobilePay Online is the fast and simple way of
```

Reply:
On second thoughts, I'm uncertain if they MobilePay or MobilePay Online in running text. I'll see if I find a discussion regarding this somewhere on Slack.